domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/Dependant.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.cmc.domain.models.statementofmeans;
+
+import lombok.Builder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.Objects;
+import java.util.Optional;
+import javax.validation.Valid;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+public class Dependant {
+
+    @Valid
+    private final Children children;
```

Comment:
I could suggest a slight variation to model this:

```
private final List<Children> children;
```

```
public class Child {

    public enum AgeGroupType {
        UNDER_11,
        BETWEEN_11_AND_15,
        BETWEEN_16_AND_19;
    }

    private final AgeGroupType ageGroupType;
    private final Integer howMany;

}
```

It could be easier and more explicit to enforce the number of children is mandatory for each age group.

Reply:
agree